src/server/index.js

```diff
@@ -131,9 +131,15 @@ app.prepare()
         logger.info("adding the api router to the express server");
         server.use("/api", api);
 
+        //map root to Dashboard
+        server.get("/", (req, res) => {
+            app.render(req, res, "/dashboard", undefined);
+        });
```

Comment:
This feels like routing logic that should be in the UI instead of the server. Maybe doing something like this somewhere? 

https://stackoverflow.com/a/58182678

Reply:
Good catch 👍 . With express, the actual doesn't get updated so I wasn't really happy it anyway.